_observing-your-data/prometheusmetrics.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: default
+title: Prometheus and Metrics Analytics
+nav_order: 40
+redirect_from:
+  - /observing-your-data/prometheusmetrics/
+---
+
+# Prometheus and Metrics Analytics
+Introduced 2.4
+{: .label .label-purple }
+
+Starting with OpenSearch 2.4, you can ingest and visualize metrics data from the open-source [Prometheus](https://prometheus.io/) monitoring solution and from log data aggregated within OpenSearch, allowing you to analyze and correlate data across logs, traces, and metrics. Previously, you could ingest and visualize only logs and traces from your monitored environments. With this feature, you can observe your digital assets with more granularity, gain deeper insight into the health of your infrastructure, and better inform your root cause analysis.
+
+The following image shows the process of pulling in metrics and viewing them in **Observability**.
```

Comment:
Can we say something like, ".... shows the process of pulling in metrics ingested through Prometheus and visualizing them in a dashboard." That way we get away from using Observability as a thing that we have to qualify.